src/pages/Landing/components/Body/ChatBox/Input/Input.module.scss

```diff
@@ -0,0 +1,29 @@
+.Input {
+    padding: 1.6rem 1.3rem;
+    height: 7.2rem;
+    background: #ebe7f3;
+    display: flex;
+    align-items: center;
+
+    img:first-child {
+        margin-right: .8rem;
+    }
+
+    img:nth-child(2) {
+        margin-right: 1.6rem;
+    }
```

Comment:
same. reusable comps

Reply:
Which component needs to reusable? please clarify it